os_stub/cryptlib_mbedtls/pk/x509.c

```diff
@@ -1702,6 +1703,8 @@ bool libspdm_set_attribute_for_req(mbedtls_x509write_csr *req, uint8_t *req_info
  * @param[in]      requester_info        requester info to gen CSR
  * @param[in]      requester_info_length The len of requester info
  *
+ * @param[in]       is_ca                if ture,set basic_constraints ture; Otherwise, set to false.
```

Comment:
here change to `if ture, set basic_constraints CA: ture`,  more accurate.

Reply:
Thanks. Changed.